wtxmgr/tx.go

```diff
@@ -12,11 +14,21 @@ import (
 	"github.com/Qitmeer/qitmeer/params"
 )
 
+var Coins = map[types.CoinID]string{
```

Comment:
You can use `types.CoinID.Name()` directly here, and there is no need to establish a mapping.